src/main/java/core/basesyntax/FigureDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDraw {
```

Comment:
Consider renaming the interface to 'Drawable' to adhere to the naming convention of using an adjective for interfaces 📝